deploy/gk-deploy

```diff
@@ -308,29 +330,29 @@ fi
 
 heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY}
 heketi-cli -s http://${heketi_service} setup-openshift-heketi-storage
-kubectl create -f heketi-storage.json
+${CLI} create -f heketi-storage.json
 
 check_pods "job-name=heketi-storage-copy-job" "Completed"
 
-kubectl delete all,service,jobs,deployment,secret --selector="deploy-heketi"
+${CLI} delete all,service,jobs,deployment,secret --selector="deploy-heketi"
 
-if [ "${CLI}" == "oc" ]; then
-  oc process heketi -v \
+if [ "${CLI}" == *oc* ]; then
+  ${CLI} process heketi -v \
          HEKETI_KUBE_NAMESPACE=heketi \
          HEKETI_KUBE_INSECURE=y \
          HEEKTI_KUBE_SECRETNAME="${heketi_secret}" \
-         HEKETI_KUBE_APIHOST="${kubeapi}" | oc create -f -
+         HEKETI_KUBE_APIHOST="${kubeapi}" | ${CLI} create -f -
```

Comment:
Constants are mainly just conventions in bash scripts. There's no way to hard code a constant value for this, as it would be different for each environment. if you mean you want the variables to be in all uppercase, that's a cosmetic issue I don't feel a need to address, as I reserve that for values that are non-generated or have static defaults (see the top of file).